### PR TITLE
updated to reflect the new ui

### LINKED ISSUE #1500: Managing Permissions chapter
### Description

Update this Authorization Services Guide chapter for the new Admin Console

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by edewit**
fixes: #1500

### CODE REVIEW

File: authorization_services/topics/permission-create-scope.adoc [hunk 1]
```diff
@@ -3,7 +3,12 @@
 
 A scope-based permission defines a set of one or more scopes to protect using a set of one or more authorization policies. Unlike resource-based permissions, you can use this permission type to create permissions not only for a resource, but also for the scopes associated with it, providing more granularity when defining the permissions that govern your resources and the actions that can be performed on them.
 
+ifeval::[{project_community}==true]
+To create a new scope-based permission, select *Create scope-based permission* from the *Create permission* dropdown.
+endif::[]
+ifeval::[{project_product}==true]
 To create a new scope-based permission, select *Scope-based* in the item list in the upper right corner of the permission listing.
+endif::[]
```
Comment (andymunro) [inline authorization_services/topics/permission-create-scope.adoc:12]: "Apply Policy" on line 35 needs to be updated.   Now I see "Policy" instead.  I don't know if it is simply a name change.

File: authorization_services/topics/permission-create-resource.adoc [hunk 2]
```diff
@@ -3,7 +3,12 @@
 
 A resource-based permission defines a set of one or more resources to protect using a set of one or more authorization policies.
 
+ifeval::[{project_community}==true]
+To create a new resource-based permission, select *Create resource-based permission* from the *Create permission* dropdown.
+endif::[]
+ifeval::[{project_product}==true]
 To create a new resource-based permission, select *Resource-based* in the item list in the upper right corner of the permission listing.
+endif::[]
```
Comment (andymunro) [inline authorization_services/topics/permission-create-resource.adoc:12]: "Apply Policy" on line 40 needs to be updated. Now I see "Policy" instead. I don't know if it is simply a name change.

**Review by andymunro — CHANGES_REQUESTED** [review]
Thanks, @edewit.  Just two comments.

**Review by andymunro — APPROVED** [review]
Thanks, @edewit.  Ready to merge, @ssilvert

**Review by ssilvert — APPROVED** [review]

**Comment by edewit** [comment]
@ssilvert merged into `main`? shouldn't it be merged to `new-admin-console`

**Comment by andymunro** [comment]
yes, @edewit, @ssilvert should be merged to new-admin-console.  If it is not possible to back this out, we can figure out if this is actually a problem.

**Comment by ssilvert** [comment]
@edewit @andymunro I didn't check to see that the PR was opened against main.

@edewit I have now reverted.  Can you open a new one against `new-admin-console`?

**Pull request merged**